src/drivers/boards/px4io-v2/board_config.h

```diff
@@ -80,6 +80,7 @@
 /* Safety switch button *******************************************************/
 
 #define GPIO_BTN_SAFETY (GPIO_INPUT|GPIO_CNF_INFLOAT|GPIO_MODE_INPUT|GPIO_PORTB|GPIO_PIN5)
+#define GPIO_SAFETY_PD  (GPIO_INPUT|GPIO_CNF_INPULLDWN|GPIO_MODE_INPUT|GPIO_PORTB|GPIO_PIN6)
```

Comment:
I am not clear on this change.  On the latest print I have 
![image](https://cloud.githubusercontent.com/assets/1945821/24718496/ef95ec96-19d1-11e7-84bb-ba405f0e9fc2.png)

Are you shorting Pin 41-42 on some version of the HW? Skype?